### PR TITLE
Update all the response format (issue #64)

### LINKED ISSUE #64: Update response format (push/update/delete docs or update/delete settings)
After pushing/updating/deleting documents or updating/resetting documents, the answer contains additional information

v0.19.0

```jsx
{
    "updateId": 0
}
```

Alpha:

```jsx
{
  "status": "pending",
  "updateId": 0,
  "meta": {
    "type": "DocumentsAddition",
    "method": "UpdateDocuments",
    "format": "Json",
    "primary_key": null
  },
  "enqueuedAt": "2021-03-02T14:37:31.345931100Z"
}
```

Should we keep all this information?

EDIT: we want to come back to the previous behavior (v0.19.0)

### PR DESCRIPTION
**Pull request by irevoire**
closes #64

### CODE REVIEW

**Review by curquiza — DISMISSED** [review]
Nice to have completed the tests!! thanks!

**Comment by curquiza** [comment]
(always wait for Marin's review before merging even if I approved)

**Comment by MarinPostma** [comment]
I think `settings` routes should also be part of this pr

**Review by MarinPostma — CHANGES_REQUESTED** [review]
It probably wasn't mentionned in the issue, but the same change needs to be applied to the setting routes. Other than that it's all good :)

File: meilisearch-http/src/index/updates.rs
Comment (irevoire) [inline meilisearch-http/src/index/updates.rs:48]: What should we do here?
There is some fields we don't implement currently:
https://docs.meilisearch.com/reference/api/settings.html#get-settings

**Comment by curquiza** [comment]
@irevoire can rebase the branch and fix git conflicts please? 🙂

File: meilisearch-http/src/index/updates.rs
Comment (curquiza) [reply meilisearch-http/src/index/updates.rs:48]: Is it related to this PR? We only want to update the body response of the updates

**Review by MarinPostma — CHANGES_REQUESTED** [review]
Hey @irevoire, it's missing the update status in the route generated in the macro @ https://github.com/meilisearch/transplant/pull/104/files#R9 :)

**Review by MarinPostma — APPROVED** [review]
good job

bors merge

**Pull request merged**
